lib/components/artist/shows/index.js

```diff
@@ -44,11 +44,11 @@ class Shows extends React.Component {
 
   currentAndUpcomingList() {
     if (this.props.artist.current_shows.length || this.props.artist.upcoming_shows.length) {
+      const shows = [].concat.apply([], [this.props.artist.current_shows, this.props.artist.upcoming_shows]);
```

Comment:
If they're in separate lists, the wrapping is off and the view will start a new row for upcoming shows. This way, we have one container for all Current & Upcoming shows.